controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -1765,3 +1780,19 @@ func (r *BareMetalHostReconciler) reconciletHostData(ctx context.Context, host *
 	return ctrl.Result{}, nil
 
 }
+
+func (r *BareMetalHostReconciler) ignoreErrorWhenCleaningModeDisabled(err error, info *reconcileInfo) bool {
+	// forbidden error can be considered harmless when CleaningModeDisabled and ns terminating
+	if k8serrors.IsForbidden(err) {
+		namespace := &corev1.Namespace{}
+		err := r.Get(context.TODO(), client.ObjectKey{Name: info.host.Namespace}, namespace)
```

Comment:
per other comments I don't think we need this, but if it is added then we need to record the RBAC permissions in the kubebuilder comments on ll. 92-103 to generate the correct RBAC manifest upstream.